pyout/tabular.py

```diff
@@ -388,15 +402,27 @@ def __call__(self, row, style=None):
             via `columns`.
 
             Instead of a plain value, a column's value can be a tuple
-            of the form (initial_value, callable).  The callable will
-            be called asynchronously with no arguments and should
-            return the value with which to replace `initial_value`.
-
-            Using callable values requires some additional steps.  The
-            `ids` property should be set unless the first column
-            happens to be a suitable id.  The instance should also be
-            used as a context manager so that the program waits at the
-            end of the block for the return values.
+            of the form (initial_value, producer).  If a producer is
+            is a generator function or generator object, each item
+            procuced replaces `initial_value`.  Otherwise, a producer
```

Comment:
Minor spelling
a generator object
produced

Reply:
"a generator object", is this just adding a "a"? In that case, I think either is ok, since it shares the "a" with "generator function", but will add if you think it's clearer.

and you're not familiar with the word "procuced"? :)

Reply:
articles -- I am not a native speaker so I could be/often is wrong. Here I think it would have been ok to have a single "a" if it was "a X or Y OBJECT", i.e. "a" would apply to the single thing of different "qualities".  But you have "a X object1 or Y object2" where object2 could be some specific one (thus might even warrant "the" applied to object2). but overall I do not really care ;)

I am indeed not familiar with "procuced" (or "procuce") and couldn't locate it in any dictionary on a quick lookup, please enlighten me

Reply:
> I am indeed not familiar with "procuced" (or "procuce") and couldn't locate it in any dictionary on a quick lookup, please enlighten me

sorry, my remark was just an unclever joke

Reply:
I did fear that it was my lack of sense of humor. QED